### ISSUE TITLE
List of custom components

### ISSUE THREAD

**Issue by maxxyouu** [IBM/carbon-ui-builder#289]
### Issue Context

angular bootstrap components

- [x] Alert
- [x] Carousel (low priority) Thursday
- [x] Collapse
- [x] Datepicker (low priority) Friday
- [x] Dropdown
- [ ] Modal (we do not have the features for it because it links to other component)
- [x] Nav
- [ ] Offcanvas (we do not have the features for it because it links to other component)
- [x] Pagination (Tuesday)
- [x] Popover
- [x] Progressbar
- [x] Rating (Monday)
- [ ] Scrollspy (not a component, therefore, we are not implement it)
- [ ] Table (too complex and we have many questions to be answered before we can implement it)
- [x] TImepicker (Monday) Thursday
- [x] Toast (Wed)
- [x] Tooltip
- [ ] Typeahead (not a component, it is a directive, therefore, we are not implement it)

**Comment by zvonimirfras** [comment]
AFAIK we can close this because it's available now as featured custom components.

Reopen if needed.
